src/DynamoCore/Extensions/StartupParams.cs

```diff
@@ -22,5 +25,10 @@ public class StartupParams
         //
         // It is expected that this class will be extended in the future, so it should stay as minimal as possible.
         //
+        public IAuthProvider AuthProvider { get; set; }
+
+        public IPathManager PathManager { get; set; }
+
+        public CustomNodeManager CustomNodeManager { get; set; }
```

Comment:
Let's expose this as an interface, `ICustomNodeManager`, rather than a concrete class.  "ExtractInterface" should be our friend here.  

All of these properties should be immutable `{ get; }`